frontend/src/views/dashboard/index.js

```diff
@@ -0,0 +1,55 @@
+import React, { Component, PropTypes } from 'react'
+import { connect } from 'react-redux'
+
+import Badge from 'components/badge'
+import { fetchFailureCount } from 'actions/dashboard/failure-count'
+import { style } from 'views/style'
+
+export class Dashboard extends Component {
+  static get propTypes () {
+    return {
+      fetchFailureCount: PropTypes.func.isRequired,
+
+      xhrStatus: PropTypes.shape({
+        isFetchingFailureCount: PropTypes.bool
+      })
+    }
+  }
+
+  componentDidMount () {
+    !this.props.failureCount &&
```

Comment:
I would prefer for the dashboard to be long-pooling for the failure count. 
This way we don't have to refresh the page by browser extensions, etc in the monitors.
(and this is super easy to achieve w/ setInterval =P)